csharp/ql/lib/semmle/code/csharp/frameworks/system/Text.qll

```diff
@@ -24,6 +25,97 @@ class SystemTextStringBuilderClass extends SystemTextClass {
   Method getAppendFormatMethod() { result = this.getAMethod("AppendFormat") }
 }
 
+/** Data flow for `System.Text.StringBuilder`. */
+private class SystemTextStringBuilderFlowModelCsv extends SummaryModelCsv {
```

Comment:
Add a `private import semmle.code.csharp.frameworks.System` to the module `Frameworks` in `ExternalFlow.qll` to ensure bidirectional importing.

Reply:
(Comment above updated)

Reply:
Will fix, but why is it needed?

Reply:
The flow summaries are compiled down to data flow nodes and atomic data-flow steps, which is a cached computation, so we want to make sure that this cached computation knows all flow summaries. If not, we may end up having to recompute this expensive computation.